build.gradle

```diff
@@ -6,7 +6,7 @@ buildscript {
         google()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.5.2'
+        classpath 'com.android.tools.build:gradle:3.5.3'
```

Comment:
Don't bump version number of  dependencies in this commit. Create a different PR for that.

Reply:
Oh lol yeah i forgot i did that srry ill change